src/jmh/java/com/trivago/kangaroo/AbstractBenchHelper.java

```diff
@@ -0,0 +1,65 @@
+package com.trivago.kangaroo;
+
+import com.trivago.fastutilconcurrentwrapper.ConcurrentLongLongMapBuilder;
+import com.trivago.fastutilconcurrentwrapper.LongLongMap;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public abstract class AbstractBenchHelper extends AbstractCommonBenchHelper {
+
+    protected static final int NUM_VALUES = 1_000_000;
+
+    protected LongLongMap map;
+
+    public void initAndLoadData(ConcurrentLongLongMapBuilder.MapMode mode) {
+        if (mode.equals(ConcurrentLongLongMapBuilder.MapMode.BUSY_WAITING)) {
+            map = ConcurrentLongLongMapBuilder.newBuilder()
+                    .withBuckets(16)
+                    .withInitialCapacity(5_000_000)
```

Comment:
* should `initialCapacity` be equal to `NUM_VALUES`

Reply:
done